express-get-json/index.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express');
+const app = express();
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+app.use((req, res, next) => {
+  const data = [];
+  for (const key in grades) {
+    data.push(grades[key]);
+  }
+  res.json(data);
```

Comment:
This should be directly inside the GET route handler for `/api/grades`, there is no need to use middleware here. We only want to build the grades array and send it to the client when a client makes a specific request to `GET /api/grades`